### DISCUSSION TITLE
Clear Linux plot time

### DISCUSSION THREAD

**Discussion by vladaepro** [madMAx43v3r/chia-plotter#692] [General]
When i switched from Ubuntu 20.04 to Clear Linux i got about 10-15% faster plot time. But i still see that majority of users here are using Ubuntu (and 10% is not small difference to waste), am i missing something, or Ubuntu 21.04 is also faster than 20.04?

**Comment by ataa** [comment]
On my low end machine, 21.04 is faster than 20.04 (about 10%). I tried to install ClearLinux while ago, but my old cpu is not supported. 
`[FAIL] FATAL: Detected Missing Required CPU features: Carry-less Multiplication extensions (pclmulqdq)`

**Reply by gryan315** [reply]
Arch linux has pretty good performance as well. I've tested photogrammetry on both Arch and Clear, and they perform about the same for that.
